README.md

```diff
@@ -1,99 +1,78 @@
 # Data Submission Service API
 
-This project can be run locally in two ways - with Docker or without. For
-speed, we recommend running _without_ Docker.
-
 Various parts of the system are documented in `/docs` and also as part of the
 [Service Manual for the service](https://github.com/Crown-Commercial-Service/ReportMI-service-manual).
 
-## Running with Docker
-
-See [Running with Docker](docs/running-with-docker.md)
-
-## Without Docker
+---
 
 ### Prerequisites
 
-Use Homebrew to install Postgres, Redis, curl, Python and csvkit on MacOS:
-
-`$ brew install postgres redis curl csvkit`
-
-The application requires a user named `postgres` so create that user if it does not exist already:
-
-`$ createuser postgres`
-
-#### Auth0 Account
-
-Liaise with the team members to get an account on Auth0. Once you have this, log into Auth0 and visit `Applications` to get the `AUTH0_CLIENT_ID` and `AUTH0_CLIENT_SECRET` values for the `.env` files (see next section)
-
-#### Yarn
-
-Install Yarn on MacOS with Homebrew:
-
-`$ brew install yarn`
-
-#### Bundler
-
-Install Bundler:
-
-`$ gem install bundler`
+- [Docker](https://docs.docker.com/docker-for-mac) greater than or equal to `18.03.1-ce-mac64 (24245)`
+- Access to the dxw 1Password vault named `DSS` can be granted by anyone on the dxw technical operations team
 
 ### Setting up the project
 
-Copy `.env.development.example` to `.env.development`. This file contains some
-secrets which need to be set. Liaise with team members to get the required
-values. See [environment-variables.md](/docs/environment-variables.md) for a
-description of all these environment variables.
-
-Once this is done, use Bundler to set up the project:
-
-`$ bundle install`
-
-Create & set up the database:
+Copy the example environment variable file used with Docker:
 
-`$ bundle exec rake db:setup`
+```bash
+$ cp docker-compose.env.sample docker-compose.env
```

Comment:
What is the purpose of this step - copying the sample file? Would it not be easier to just say that the contents of the file from 1Password should be saved to a file called `<repository-root>/docker-compose.env`?

Reply:
The intent with the copy command is to provide a small step which gets the application booting successfully, which it should be able to do with nothing more than presence of keys with blank values. 

There is not enough value to hang my hat here on so don't hold this view very strongly!

The value of the approach is perhaps more evident on other projects where 1password was not immediately available and was a blocker to anyone new getting it running whilst their access to whatever secret tool was sorted. Some examples of other repos where we've taken this approach  https://github.com/dfe-digital/teacher-vacancy-service#setting-up-the-project, https://github.com/dxw/mytvh-web#application-setup, https://github.com/dxw/ukti-opportunities#installation, https://github.com/dxw/golden_retriever#installation